client/src/scenes/PrivacyPolicy.jsx

```diff
@@ -16,20 +16,20 @@ export default function PrivacyPolicy() {
                     <p style={{'font-size': '18px'}}>
                         מעירים (ע.ר.) היא המפתחת והמפעילה של הפלטפורמה, לרבות האתר ושאר השירותים המסופקים על-ידה (כולם ביחד – <b>"הפלטפורמה"</b>).
                     </p>
-                    <p>
+                    <p style={{'font-size': '18px'}}>
```

Comment:
please add `p` styles as a children of `container` (or give this an id), very repetitive (all p's that are children here get the same style of font-size:18px)